components/automate-chef-io/content/docs/restore.md

```diff
@@ -0,0 +1,119 @@
+         fqdn = "<new-fqdn>"
+
+         # To provide a cert and key for the restore host, uncomment and fill
+         # these sections.
+         # [[global.v1.frontend_tls]]
+         # The TLS certificate for the load balancer frontend.
+         # cert = """-----BEGIN CERTIFICATE-----
+         # <certificate-for-new-fqdn>
+         # -----END CERTIFICATE-----
+         # """
+
+         # The TLS RSA key for the load balancer frontend.
+         # key = """-----BEGIN RSA PRIVATE KEY-----
+         # <key-for-new-fqdn>
+         # -----END RSA PRIVATE KEY-----
+         # """
+    ```
+
+## Restore From a Filesystem Backup
+
+Meet the required [prerequisites]({{< ref "restore.md#prerequisites" >}}) before beginning your restore process.
+
+### Restore in an Internet-Connected Environment
+
+If you have [configured the backup directory]({{< relref "backup.md#backup-to-a-filesystem" >}}) to a directory other than the default directory (`/var/opt/chef-automate/backups`), you must supply the backup directory.
+Without a backup ID, Chef Automate uses the most recent backup in the backup directory.
+
+```shell
+chef-automate backup restore </path/to/backups/>BACKUP_ID [--patch-config </path/to/patch.toml>] [--skip-preflight]
+```
+
+Use the `--patch-config` option with a [configuration patch file]({{< relref "backup.md#prerequisites" >}}) to restore to a host with a different FQDN than that of the backup host.
+Use the `--skip-preflight` option to restore to a host with a pre-existing Chef Automate installation.
+
+Incorrect directory permissions cause restore from a filesystem backup to fail.
+Run the [`fix-repo-permissions` command]({{< ref "cli-chef-automate/#chef-automate-backup-fix-repo-permissions" >}}) to address such issues:
+
+```shell
+sudo chef-automate backup fix-repo-permissions <path>
+```
+
+### Restore in an Airgapped Environment
+
+To restore a backup of an [airgapped installation]({{< relref "airgapped-installation.md" >}}), you must specify the [Airgap Installation Bundle]({{< relref "airgapped-installation.md#create-an-airgap-installation-bundle" >}}) used by the installation.
+If you have [configured the backup directory]({{< relref "backup.md#backup-to-a-filesystem" >}}) to a directory that is not the default `/var/opt/chef-automate/backups`, then you must supply the backup directory.
+If you do not provide a backup ID, Chef Automate uses the most recent backup in the backup directory.
+
+```shell
+chef-automate backup restore --airgap-bundle </path/to/bundle> </path/to/backups/>BACKUP_ID [--patch-config </path/to/patch.toml>] [--skip-preflight]
+```
+
+Use the `--patch-config` option with a [configuration patch file]({{< relref "backup.md#prerequisites" >}}) to restore to a host with a different FQDN than that of the backup host.
+Use the `--skip-preflight` option to restore to a host with a pre-existing Chef Automate installation.
+
+Incorrect directory permissions cause restore from a filesystem backups to fail.
```

Comment:
Awkward sentence, please revise (used 3x in this PR)

Reply:
Fixed!